pkgs/test_core/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.6.14
+* Support environment flags for configuring the global declarer.
```

Comment:
This reads like we support reading from `Platform.environment`.

I wonder, _should_ we be trying to do that instead? For the VM case it's probably little consequence either way, for AOT native we'd probably prefer `Platform.environment` support, and for web we'd prefer what's implemented.

Would there be value in supporting a `Platform.environment` fallback in places where we can import it? I'd be happy to write up the conditional import version if we think it's useful.